requirements.txt

```diff
@@ -0,0 +1,10 @@
+pytest
```

Comment:
2 comments:

- we shouldn't mix both runtime and test dependencies here, please move test dependencies to a separate `requirements.txt` file (it can live in `tests/` directory)
- we want to lock down versions of our runtime dependencies - we use `pip-compile` for that purpose in other repositories. Please follow the same practice here (add `requirements.in` file, https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/requirements.in and generate `requirements.txt` from it using `pip-compile`).